DEPLOYMENT_GUIDE.md

```diff
@@ -239,7 +231,7 @@ Services are organized into profiles:
 - **download**: Download-related services (API, Worker)
 - **rabbit**: RabbitMQ message queue
 
-### Multinetwork Gateway
+### Multinetwork Gateway (retired)
```

Comment:
Can this section just be deleted instead?

Reply:
Jeremy asked me to keep it